go_utils/photo_download.py

```diff
@@ -48,11 +54,22 @@ def download_photo(url, directory, filename):
     filename : str
         The name of the photo
     """
-    downloaded_obj = requests.get(url, allow_redirects=True)
-    filename = remove_bad_characters(filename)
-    out_path = os.path.join(directory, filename)
-    with open(out_path, "wb") as file:
-        file.write(downloaded_obj.content)
+    if any(x is not None for x in [url, directory, filename]):
```

Comment:
I think for nonetype checking, it might be more robust to use `pd.isna()` (applies to other usages if None)
```suggestion
    if any(not pd.isna(x) for x in [url, directory, filename]):
```

Reply:
done! -- Thanks for the tip!